hoomd/md/pytest/test_methods.py

```diff
@@ -67,6 +67,91 @@ def test_brownian_attributes_attached(simulation_factory,
     brownian.alpha = 0.125
     assert brownian.alpha == 0.125
 
+def test_brownian_rattle_attributes():
+    """Test attributes of the Brownian RATTLE integrator before attaching."""
+    all_ = hoomd.filter.All()
+    constant = hoomd.variant.Constant(2.0)
+    gyroid = hoomd.md.manifold.Gyroid(N=1)
+    brownian_rattle = hoomd.md.methods.Brownian_Rattle(filter = all_, kT=constant, manifold = gyroid, seed=2, eta = 1e-5)
+
+    assert brownian_rattle.filter is all_
+    assert brownian_rattle.kT is constant
+    assert brownian_rattle.manifold is gyroid
+    assert brownian_rattle.seed == 2
+    assert brownian_rattle.alpha is None
+    assert brownian_rattle.eta == 1e-5
+
+    type_A = hoomd.filter.Type(['A'])
+    brownian_rattle.filter = type_A
+    assert brownian_rattle.filter is type_A
+
+    ramp = hoomd.variant.Ramp(1, 2, 1000000, 2000000)
+    brownian_rattle.kT = ramp
+    assert brownian_rattle.kT is ramp
+
+    sphere = hoomd.md.manifold.Sphere(r=10)
+    brownian_rattle.manifold = sphere
+    assert brownian_rattle.manifold is sphere
+
+    brownian_rattle.seed = 10
+    assert brownian_rattle.seed == 10
+
+    brownian_rattle.eta = 0.001
+    assert brownian_rattle.eta == 0.001
+
+    brownian_rattle.alpha = 0.125
+    assert brownian_rattle.alpha == 0.125
+
+
+def test_brownian_rattle_attributes_attached(simulation_factory,
+                                      two_particle_snapshot_factory):
+    """Test attributes of the Brownian RATTLE integrator after attaching."""
+    all_ = hoomd.filter.All()
+    constant = hoomd.variant.Constant(2.0)
+    plane = hoomd.md.manifold.Plane()
+    brownian_rattle = hoomd.md.methods.Brownian_Rattle(filter = all_, kT=constant, manifold = plane, seed=2)
+
+    sim = simulation_factory(two_particle_snapshot_factory())
+    sim.operations.integrator = hoomd.md.Integrator(0.005, methods=[brownian_rattle])
+    sim.operations._schedule()
```

Comment:
I believe the preferred way to do this now is just 
```suggestion
    sim.run(0)
```